service/dap/server.go

```diff
@@ -1950,6 +1969,12 @@ func (s *Session) onAttachRequest(request *dap.AttachRequest) {
 	}
 	s.config.log.Debug("parsed launch config: ", prettyPrint(args))
 
+	if s.args.followExec {
+		s.sendShowUserErrorResponse(request.Request, FailedToAttach, "Failed to attach",
+			"Follow exec not supported in attach request yet.")
+		return
+	}
```

Comment:
Is this a critical error? Is it better UX for the session to fail outright, or for it to continue as best it can? As I said above, my inclination would be to continue.